server/services/gameLogicClass.js

```diff
@@ -165,6 +184,23 @@ module.exports = class GameLogic {
             throw Error("You cannot vote for a card more than once, or now is not the right time to vote.");
         }
     }
+    
+    emitVotes() {
+        this.clearTimeouts();
+        this.status = statusTypes.DISPLAY_ALL_VOTES;
+        socket.emitStatus(this.roomId, this.status);
+        socket.emitAllVotes(this.roomId, this.votes);
+        this.calcScores();
+        this.nextRoundTimeout = setTimeout(() => this.nextRound(), this.timeoutDuration);
```

Comment:
```suggestion
        this.nextRoundTimeout = setTimeout(this.nextRound, this.timeoutDuration);
```

Reply:
Also, I think this file should be renamed: `gameLogicClass` -> `GameLogic`. Shouldn't have "class" in the name.

Reply:
I think if you just do this.nextRound it invokes that funciton straight away @Ro4052